blogposts/gophercon-2019-detecting-incompatible-api-changes.md

```diff
@@ -336,9 +336,9 @@ We’ve talked about defined types. Let’s move on to channels.
 
 **Channels**
 
-There’s only one compatible change you can make to a channel type: you can remove the direction marker. 
+there's only one compatible change you can make to a channel type: you can remove the direction marker. 
 
-For example, if you started with a channel that you can only send on, you can change that to a channel that can either send or receive. That’s clearly safe, because any code that tried to receive from the channel wouldn’t have compiled in the first place.
+For example, if you started with a channel that you can only send on, you can change that to a channel that can either send or receive. That’s clearly safe, because any code that tried to receive from the channel Wouldn't have compiled in the first place.
```

Comment:
```suggestion
For example, if you started with a channel that you can only send on, you can change that to a channel that can either send or receive. That’s clearly safe, because any code that tried to receive from the channel wouldn't have compiled in the first place.
```